cmd/dep/init_test.go

```diff
@@ -144,3 +147,125 @@ const Qux = "yo yo!"
 	// vendor should have been created & populated
 	h.MustExist(h.Path("src/" + root + "/vendor/github.com/Sirupsen/logrus"))
 }
+
+// TestCaseInsentitive is test for Windows. This should work even though set
+// difference letter cases in GOPATH.
+func TestCaseInsentitive(t *testing.T) {
```

Comment:
This is adding a fair bit of logic and test running time for what seems to me like a fairly straightforward check - can we make something more like a unit test?